src/containers/RegisterForm.tsx

```diff
@@ -1,9 +1,21 @@
 import { connect } from 'react-redux';
 import { withFormik } from 'formik';
+import * as Yup from 'yup';
 
 import RegisterForm from '../components/RegisterForm';
 import { registerAccount } from '../actions/registerActions';
 
+const RegisterSchema = Yup.object().shape({
+  firstName: Yup.string().max(20, 'Too Long').required('Required'),
+  lastName: Yup.string().max(20, 'Too Long').required('Required'),
+  email: Yup.string().email('Invalid Email').required('Required'),
+  password: Yup.string().min(8, 'Too Short').required('Required'),
```

Comment:
Min pass word should be 10 not 8 - a 9 length password will throw a random error

Reply:
Not quite sure what you mean that 9 length gives an error. I just tried it and it worked for sign up and sign in. I'll bump the min up to 9 but I don't think 10 is necessary.